plugins/tmux-autoattach/tmux-autoattach.plugin.sh

```diff
@@ -14,7 +14,7 @@ _osh_plugin_tmux_autoattach_detach() {
 	[ -z "$TMUX" ] && tmux -2u new -A
 }
 
-if [ "$DISPLAY" ] || [ "$WAYLAND_DISPLAY" ]; then
+if [ "$DISPLAY" ] || [ "$WAYLAND_DISPLAY" ] || [ "$SSH_TTY" ]; then
```

Comment:
Also added SSH. These checks are mostly here to ensure this doesn't run on a console. I typically login using xinitrc, so this works for me. But if this becomes important, maybe we add another variable that could be used to ignore these checks.

Reply:
I was about to write that we should make it configurable thinking that other users will be confused by the fact that it won't attach to `tmux` just by enabling the plugin, ...

But wait... such an option would just turn on/off the plugin actually. Then, I think we can just write

```bash
[ "$SSH_TTY" ] && plugins+=(tmux-autoattach)
```

at the user's side if the user would like to require conditions to turn on the plugin. We can optionally write about this type of setup in `tmux-autoattach/README.md` so that the users can specify any conditions for turning on the plugin on their own.

Reply:
This is an interesting idea. I assume we could generalize the application a little more so that this format could be used with any plugin and mention something in the larger OSH documentation. The individual plugin documentation could have a blurb about specific use cases for such conditionals.

I'm going to remove all checks and make note of this in the documentation.

Reply:
Ah, yes. That's a good idea. Please go ahead and update the `oh-my-bash` documentation! Thank you!